pkg/api/batch-submitter.go

```diff
@@ -0,0 +1,71 @@
+package api
+
+import (
+	"fmt"
+
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/kubernetes"
+	"github.com/tokamak-network/tokamak-titond-backend/pkg/model"
+)
+
+func (t *TitondAPI) CreateBatchSubmitter(bs *model.Component, config *ComponentConfig) (*model.Component, error) {
```

Comment:
Great!
Likewise, We can delete config now(PoC) in here. Everything else except this seems great to me! 😍

Reply:
Thanks!